src/App.tsx

```diff
@@ -37,6 +39,16 @@ function App() {
                 throw new Error(message);
             }
             const data = await response.json();
```

Comment:
In order to get the type right, we need to assign the right data type here and then modify the fields as necessary to get all the red to go away. This data variable should have type `TodoItemResponse` - it's the data you get back from Airtable when you make the request. You should then modify that type so that it agrees with this function you've written.

Normally I would leave you some hints here, but because the class is almost over this is what I would recommend you change the TodoItemResponse type to be:

```
//what I get back from airtable
type TodoItemResponse = {
	records: TodoRecord[];
};

type TodoRecord = {
	fields: {
		title: string;
	};
	id: number;
};
```

Then you'll need to change your objectA and objectB below to have type TodoRecord. They aren't purely TodoItem types because those don't have `.field` - it's just a type you use locally in your app and is different than how Airtable represents them, hence creating a separate type for the response here `TodoRecord`, which organizes them in a different way.